app/uk/gov/hmrc/apiplatform/modules/approvals/domain/services/ResponsibleIndividualVerificationFrontendJsonFormatters.scala

```diff
@@ -18,10 +18,11 @@ package uk.gov.hmrc.apiplatform.modules.approvals.domain.services
 
 import uk.gov.hmrc.apiplatform.modules.approvals.domain.models.{ResponsibleIndividualVerification, ResponsibleIndividualVerificationWithDetails}
 import play.api.libs.json._
+import uk.gov.hmrc.mongo.play.json.formats.MongoJavatimeFormats
 
 trait ResponsibleIndividualVerificationFrontendJsonFormatters extends EnvReads {
 
-  implicit val utcReads = DefaultLocalDateTimeReads
+  implicit val dateFormat = MongoJavatimeFormats.localDateTimeFormat
```

Comment:
Why not just import the localDateTimeFormat implicit instead of creating a new val?